alsa_settings/LNLM_SDW_AIOC.sh

```diff
@@ -0,0 +1,12 @@
+set -e
+
+# enable headset playback
+amixer -c sofsoundwire cset name='Headphone Switch' on
+amixer -c sofsoundwire cset name='rt711 FU05 Playback Volume' 80
+
+# enable headset capture
+amixer -c sofsoundwire cset name='Headset Mic Switch' on
+amixer -c sofsoundwire cset name='rt711 FU0F Capture Switch' on
+amixer -c sofsoundwire cset name='rt711 FU1E Capture Switch' on
+amixer -c sofsoundwire cset name='rt711 FU0F Capture Volume' 30
+amixer -c sofsoundwire cset name='rt711 FU1E Capture Volume' 30
```

Comment:
what about LNLM_SDW_RVP without AIOC? That's what I use so far :-)

Reply:
LNLM_SDW_RVP is not our target test config. We will switch all LNLM_SDW_RVP to LNLM_SDW_AIOC.